app/controllers/analytics_controller.rb

```diff
@@ -18,6 +18,8 @@ def index
     @cases_sorted_by_followers = Case.sorted_by_followers 10
     @most_visited_cases = DetermineVisitsToCases.call(@visits_this_week.sorted_by_hits(13))
     @most_recent_comments = Comment.sorted_by_creation 15
+    @case_updates_30 = PaperTrail::Version.where('created_at > ?', 30.days.ago)
```

Comment:
Hold on - if we're trying to display the last 10 editors, then why are we querying for the versions of cases created in the past 30 days?

Reply:
The query is intended to find versions (new cases or edits of old cases) created in the past 30 days. Then group by whodunnit and lastly find those users and sort them in order of how many versions each of them is responsible for.